cluster/manifests/kube-job-cleaner/cronjob.yaml

```diff
@@ -8,8 +8,13 @@ metadata:
     version: "0.3"
 spec:
   schedule: "17 * * * *"
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 1
   jobTemplate:
+    metadata:
+      creationTimestamp: null
```

Comment:
nit: Does this `creationTimestamp: null` do anything? Otherwise I would just remove it to keep it simple as this manifest could be used as example for others.

Reply:
good catch seems to be a leftover from dumping cronjob :)